.github/CODE_OF_CONDUCT.md

```diff
@@ -56,3 +56,39 @@ Original text courtesy of the Speak Up! project and Django Project.
 ## QUESTIONS?
 
 If you have questions, please see the FAQ. If that doesn’t answer your questions, feel free to email conduct@js.foundation.
+
+# OpenJS Foundation Code of Conduct
+
+The OpenJS Foundation and its member projects use the Contributor
+Covenant v1.4.1 as its Code of Conduct. Refer to the following
+for the full text:
+
+- [english](https://www.contributor-covenant.org/version/1/4/code-of-conduct)
+- [translations](https://www.contributor-covenant.org/translations)
+
+Refer to the section on reporting and escalation in this document for the specific emails that can be used to report and escalate issues.
+
+## Reporting
+
+### Project Spaces
+
+For reporting issues in spaces related to a member project please use the email provided by the project for reporting. Projects handle CoC issues related to the spaces that they maintain. Projects maintainers commit to:
```

Comment:
`please use the email provided by the project for reporting`

Have we the email?

Reply:
Good catch. 
Sounds like pointing to Gitter is not acceptable. Altho seems overkill to introduce an email just for this, but if its part of the CoC 🤷‍♂ .

@boneskull @Munter @juergba any thoughts?

Reply:
If I call a hotline to complain about a service and I never get an answer, that's just a pure provocation. Better no hotline at all, than a hotline that doesn't really care.
So an email without defining a responsible maintainer: IMO no.
Do we have resources for this: IMO no.

Reply:
I agree with no hotline. 
So, we will remove this? And can we remove this because it is OpenJS template?

Reply:
I didnt think we should remove it for that reason. I feel as a whole we do adhere to the `Projects maintainers commit to` points.

Reply:
I will be the point of contact for this for now